requirements.txt

```
@@ -4,6 +4,7 @@ click==8.0.1              # via flask
 flask-jwt-extended==3.24.1  # via patientsearch (setup.py)
 # pin to last hash of master, to remove requirement of freestanding JSON config file
 git+https://github.com/uwcirg/flask-oidc.git@ed4eb20#egg=flask-oidc
+git+https://github.com/uwcirg/fhir-migrations
```

we might want to start tagging and pinning fhir-migrations, so we're prepared when we need to make breaking changes

```suggestion
git+https://github.com/uwcirg/fhir-migrations@0.1.0
```

I went ahead and [created a tag on fhir-migrations](https://github.com/uwcirg/fhir-migrations/releases/tag/0.1.0). I followed [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) format, but feel free to make another if you have another tag format in line